Dockerfile

```diff
@@ -196,6 +196,12 @@ RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key
         libxss1 \
     && rm -rf /var/lib/apt/lists/*
 
+# Install sqlite3 - Required for cucumber-rails
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
```

Comment:
To be consistent, could you please use the long version `--assume-yes` instead of `-y`?

Reply:
This was what is in the repo. I copied that?

Reply:
Indeed, a bunch of `-y` remains 😅